Submissions/Bain_Week2.playground/Contents.swift

```diff
@@ -0,0 +1,62 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let explicitFloat: Float = 4.0
+
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width)
+/* Error message: Binary operator '+' cannot be applied to operands of type 'String' and 'Int' */
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+let feet = 5.0
+let inches = 0.7
+let myGreeting = "Hello Melissa. You are \(feet + inches) tall."
+
+//: Control Flow
+//: 1. Change optional Name to nil. What greeting do you get? Add an else clause that sets a different greeting if optionalName is nil.
+var optionalString: String? = "Hello"
+print(optionalString == nil)
+
+var optionalName: String? = "John Appleseed"
+var greeting = "Hello!"
+if let name = optionalName {
+    greeting = "Hello, \(name)"
+} else {
+    greeting = "Hey there noName!"
+}
+
+// When I change optionalName to nil, the greeting returned is Hello! - this is from the initial declaration.
+//: 2. Try removing the default case. Copy the error and place it in a block comment below this block of code.
+let vegetable = "red pepper"
+switch vegetable {
+case "celery":
+    print("Add some raisins and make ants on a log.")
+case "cucumber", "watercress":
+    print("That would make a good tea sandwich.")
+case let x where x.hasSuffix("pepper"):
+    print("Is it a spicy \(x)?")
+default:
+    print("Everything tastes good in soup.")
+}
+
+/* Error Message: Switch must be exhastive */
+//: 3. Add another variable to keep track of which kind of number was the largest, as well as what that largest number was.
+
+let interestingNumbers = [
+    "Prime": [2, 3, 5, 7, 11, 13],
+    "Fibonacci": [1, 1, 2, 3, 5, 8],
+    "Square": [1, 4, 9, 16, 25],
+]
+var largest = 0
+var largestKind = ""
+
+for (kind, numbers) in interestingNumbers {
+    for number in numbers {
+        if number > largest {
+            largest = number
+            largestKind = kind
+        }
+    }
+}
+print(largest)
+print(largest)
```

Comment:
You are printing `largest` twice. I think maybe you meant to print `largestKind`. Your implementation is otherwise correct. :-D